secure_message/resources/threads.py

```diff
@@ -21,13 +22,77 @@ def get(thread_id):
         logger.info("Getting messages from thread", thread_id=thread_id, user_uuid=g.user.user_uuid)
 
         conversation = Retriever().retrieve_thread(thread_id, g.user)
+        conversation_metadata = Retriever.retrieve_conversation_metadata(thread_id)
+        if conversation_metadata is None:
+            abort(404)
```

Comment:
This causes a http 500 error in response ops when viewing any conversations that don't have a conversation record. All old conversations (started prior to deploying this pull request) won't have a conversation record.

Reply:
Fixed this and tested it for myself.